ml_genn/ml_genn/compilers/event_prop_compiler.py

```diff
@@ -631,24 +632,12 @@ def __init__(self, example_timesteps: int, losses,
                                "{\"weight\": \"adam\"} to optimise all "
                                "weights with the adam optimiser")
         
-        # If legacy upper and lower regularisation strength are specified
-        # **NOTE** needs to be before superclass call so these get removed 
-        # from genn_kwargs as unknown arguments now causes an error
-        reg_lambda_upper = genn_kwargs.pop("reg_lambda_upper", None)
-        reg_lambda_lower = genn_kwargs.pop("reg_lambda_lower", None)
-        if reg_lambda_upper is not None or reg_lambda_lower is not None:
-            # If they match, use as regularisation strength
-            if reg_lambda_upper == reg_lambda_lower:
-                reg_lambda = reg_lambda_upper
-                warn("Seperate 'reg_lambda_upper' and 'reg_lambda_lower' "
-                     "arguments for EventPropCompiler are no longer "
-                     "supported, please use 'reg_lambda'", FutureWarning)
-            # Otherwise, give error
-            else:
-                raise NotImplemented("Seperate 'reg_lambda_upper' and "
-                                     "'reg_lambda_lower' arguments for "
-                                     "EventPropCompiler are no longer "
-                                     "supported, please use 'reg_lambda'")
+        # If reg_lambda is specified, use it for both, reg_lambda_lower and
+        # reg_lambda_upper
+        reg_lambda = genn_kwargs.pop("reg_lambda", None)
+        if reg_lambda is not None:
+            reg_lambda_lower = reg_lambda
+            reg_lambda_upper = reg_lambda
```

Comment:
```suggestion
        # If regularisation strength is specified as a single float, use for both upper and lower
        if isinstance(reg_lambda, float):
            self.reg_lambda_lower = reg_lambda
            self.reg_lambda_upper = reg_lambda
        # Otherwise, unpack
        else:
            self.reg_lambda_lower, self.reg_lambda_upper = reg_lambda
```